.circleci/config.yml

```diff
@@ -51,6 +51,13 @@ commands:
   build:
     description: "Build"
     steps:
+      - run:
+          name: "Create environment variables file"
+          command: |
+            echo GD_CLIENT_ID=${GD_CLIENT_ID} > .env
+            echo GD_API_KEY=${GD_API_KEY} >> .env
```

Comment:
What is `>` and `>>`
And what is difference?

Reply:
These are `redirection` utilities in shell that allows us to redirect standard output (`stdout`) to another stream except than terminal itself.
The first one (`>`) truncates file content and the second (`>>`) appends to a file while writing content.